benches/generated_get.rs

```diff
@@ -0,0 +1,104 @@
+use std::time::Duration;
+
+use blart::{
+    tests_common::{
+        generate_key_fixed_length, generate_key_with_prefix, generate_keys_skewed, PrefixExpansion,
+    },
+    TreeMap,
+};
+use criterion::{measurement::Measurement, BenchmarkGroup, Criterion};
+
+#[inline(always)]
```

Comment:
I feel like the usage of `#[inline(always)]` may be a bit excessive (case in point this usage and others in this file)

It definitely makes sense in some instances, but I'm seeing blanket applied over the most of the crate. Did you happen measure when you applied these annotations what the difference in runtime vs code size was?

Reply:
Yes, most of the inlines that I added were measured, some of them are unecessary, but I like to be 100% sure that the compiler is inlining some operations. In this case I don't even remember why I added them, but for the internal stuff, like in the representation and so on... I measured the difference.